src/Application/inc/Calculator.h

```diff
@@ -7,10 +7,13 @@
 * @brief calculator function declarations.
 */
 ////////////////////////////////////////////////////////////////////////////////
-
```

Comment:
Maybe keep a new line between Doxygen comments and #includes and such, just a though